spec/features/user_signs_in_spec.rb

```diff
@@ -22,7 +22,7 @@
   scenario 'specify invalid credentials' do
     visit new_user_session_path
 
-    click_button 'Log in'
+    click_button 'Sign in'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.